api/resources/translations/messages-sw.properties

```diff
@@ -1,120 +1,127 @@
-All\ contact\ types = aina ya nambari zote
-All\ facilities = mahali kote
-All\ form\ types = aina ya fomu zote
+Advanced\ settings\ outro = Hii ni ya kutumika na wale walio na ujuzi wa juu wa technologia
+All\ contact\ types = Aina ya nambari zote
+All\ facilities = Mahali kote
+All\ form\ types = Aina ya fomu zote
 Analytics = Grafu
-Antenatal\ Care = huduma wakati mama ana ujauzito
-Any\ date = tarehe yoyote
+Antenatal\ Care = Huduma wakati mama ana ujauzito
+Any\ date = Tarehe yoyote
 Any\ status = Hali ilivyo
+api.startup.checks = Ukaguzi wa ufungaji
+api.startup.config = Kutekeleza mipangilio
+api.startup.forms = Kusakinisha fomu
+api.startup.install = Kusakinisha
+api.startup.index = Kuweka data faharasa
+api.startup.migrate = Kuhamisha data
+api.startup.title = {{ branding.name }} inaanza
 Application\ Text = Lugha ya nakala
 Apply = Tumia
 Appointment\ Date = Tarehe ya kutembele kliniki
 At\ home\ with\ SBA = Kuzalia nyumbani na muuguzi
-At\ home\ without\ SBA = kuzalia nyumbani bila muuguzi
+At\ home\ without\ SBA = Kuzalia nyumbani bila muuguzi
 Audit\ Logs = Kumbukumbu ukaguzi
 Automated\ Reply = Majibu ya moja kwa moja
-Available\ Fields = sehemu inayopatikana
+Available\ Fields = Sehemu zinazopatikana
 Back = Nyuma
-Backup = nakala za dharura
-Basic = msingi
+Backup = Nakala za dharura
+Basic = Msingi
 Bug\ description = Maelezo ya hitilafu
 Cancel = Futa
 Choose\ File = Chagua Faili
-Choose\ file = changua faili
+Choose\ file = Chagua faili
 Clear\ all\ filters = Chuja upya
 Clinic = Eneo
 Clinic\ Contact\ Name = Jina
 Clinic\ Contact\ Phone = Nambari ya simu
 Clinics = Wafanyakazi wa Afya ya Jamii
 Community\ health\ worker = Daktari wa nyanjani
-Configuration = mipangilio
+Configuration = Mipangilio
 Confirm = Thibitisha
-Confirm\ Password = thibitisha neno fiche
-Confirm\ delete = una uhakika wataka kufuta {{name}}? huwezi kubadilisha baadaye
-Contact = mwasiliani
+Confirm\ Password = Thibitisha neno fiche
+Confirm\ delete = Una uhakika wataka kufuta {{name}}? huwezi kubadilisha baadaye
+Contact = Mwasiliani
 Contact\ type = Aina ya jumbe za mawasiliano
 Contacts = Wasiliani
-Contacts\ file\ help = tafuta faili ya .json iliyo na wasiliani
+Contacts\ file\ help = Tafuta faili ya .json iliyo na wasiliani
 Content = Yaliyomo
 Continue = Endelea
-Current\ Password = 
-Dashboard\ settings\ page = hii ni ukurasa wa mazingira ya dashibodi
+Current\ Password = Neno fiche la sasa
+Dashboard\ settings\ page = Hii ni ukurasa wa mazingira ya dashibodi
 Date\ display\ format = Jinsi ya kuonyesha tarehe
 Datetime\ display\ format = Jinsi ya kuonyesha tarehe na saa
 Default = Chaguo-msingi
 Default\ Application\ Language = Lugha halisi ya programu
 Default\ country\ code = Kodi ya nchi
 Default\ country\ code\ help = Kodi hii itaambatana na nambari zote za simu zitakazo wekwa kwenye programu ya Medic Mobile bila kodi ya nchi.
 Delete = Kufuta
-Deleting = kufuta...
+Deleting = Inafuta...
```

Comment:
I think this should remain as `Kufuta`. Inafuta translates to 'It is deleting' and not just deleting.